src/gettext.cpp

```diff
@@ -411,15 +411,33 @@ std::string dsgettext (const char * domainname, const char *msgid)
 	return msgval;
 }
 
+namespace {
+
+inline const char* is_unlocalized_string2(const std::string& str, const char* singular, const char* plural)
```

Comment:
Why use `inline` in a .cpp file (as opposed to in an .hpp file)?

Reply:
Personal preference. That's just my way of saying "this tiny function could as well be written into its caller but i wanted it to have a cute name" to people reading the code.